control/control.MicroOS.xml

```diff
@@ -239,6 +239,10 @@ textdomain="control"
         <id>micro_os_role</id>
 
         <order config:type="integer">100</order>
+        <software>
+            <default_patterns>microos_tools</default_patterns>
+        </software>
+
```

Comment:
This is very wrong - it would result in a role that ONLY installs microos_tools...which would be utterly and utterly broken

I think you meant to add microos_tools to the default_patterns defined on line 104

Reply:
Had a feeling. I much prefer your idea to use conditionals in the rpm spec instead of this, closing this PR.